README.md

```diff
@@ -34,35 +34,40 @@ necsim-rust is built in a modular way to reduce code duplication and allow the u
 - spatially-explicit simulation with habitat and dispersal maps
 
 ## Prerequisites
-
 First, you need to clone the necsim-rust GitHub repository:
 ```shell
-> git clone https://github.com/juntyr/necsim-rust.git
+git clone https://github.com/juntyr/necsim-rust.git
 ```
 necsim-rust is written in the [Rust language](https://www.rust-lang.org/tools/install), which must be installed in your `PATH` first. necsim-rust includes a `rust-toolchain` file that configures Rust to use a working nightly toolchain version and install all components required for compilation. If you want to use necsim-rust on a target different than `x86_64-unknown-linux-gnu`, please update the [rust-toolchain](rust-toolchain) config file accordingly.
 
 If you also want to use the CUDA-based algorithm, it is **required** that you also install the following:
 ```shell
-> cargo install ptx-linker --force --locked
+cargo install ptx-linker --force --locked
 ```
 
+Finally `rustcoalescence` depends on `libclang-dev` as well as `libsqlite3-dev` to install these packages please run:
```

Comment:
Some of the reporters depend on them, not rustcoalescence itself (and compiling the reporters is somewhat optional), so I think this should be more specific